src/stores/PreferenceStore.ts

```diff
@@ -657,7 +671,8 @@ export class PreferenceStore {
                 PreferenceKeys.REGION_CREATION_MODE,
                 PreferenceKeys.WCS_OVERLAY_AST_COLOR,
                 PreferenceKeys.CATALOG_TABLE_SEPARATOR_POSITION,
-                PreferenceKeys.PIXEL_GRID_COLOR
+                PreferenceKeys.PIXEL_GRID_COLOR,
+                PreferenceKeys.WCS_OVERLAY_CURSOR_INFO
```

Comment:
Do we also add `PERFORMANCE_LIMIT_OVERLAY_REDRAW` in `booleanKeys`?

Reply:
those are only needed for older preferences that we need to upgrade. But that means I can remove `Preferences.WCS_OVERLAY_CURSOR_INFO` from that array as well :) I've added a comment that clarifies this as well